vendor.conf

```diff
@@ -37,11 +37,18 @@ github.com/opencontainers/runc v1.0.0-rc2
 # OCI runtime spec, latest release as of 12/16/2016
 github.com/opencontainers/runtime-spec v1.0.0-rc3
 # logrus, latest release as of 12/16/2016
+# FIXME: github.com/docker/docker/pkg/archive requires this pkg as github.com/Sirupsen/logrus.
+# So we vendor github.com/Sirupsen as well.
```

Comment:
I think it's better to update `docker/docker`, the name is never going back to have a capital S.